shard-operator-parent/shard-operator-core/src/main/java/com/redhat/service/smartevents/shard/operator/core/resources/Condition.java

```diff
@@ -53,6 +53,7 @@ public class Condition {
     public Condition(@JsonProperty("type") final String type, @JsonProperty("status") final ConditionStatus status) {
         this.type = type;
         this.status = status;
+        this.lastTransitionTime = new Date();
```

Comment:
Better to use `ZonedDateTime.now(ZoneOffset.UTC);`

Reply:
done

Reply:
`lastTransitionTime`  is `Date` inside `shard-operator-core`. If I make it `ZonedDateTime` them it would require changes in v1 codebase.

Maximum I could do it add conversion here `this.lastTransitionTime = Date.from(ZonedDateTime.now(ZoneOffset.UTC).toInstant());`

Reply:
Until the conditions were not propagated to the manager, it was okaish to have a different representation of the a date in the control plane and the data plane (it was not ideal, but acceptable). Now that the Manager expects a ZonedDateTime for the Conditions, we have to align with that. Can you please check if it's possible to change the lastTransitionTime property to a `ZonedDateTime`?

Reply:
As I say `lastTransitionTime` is `Date` inside `shard-operator-core`. If I make it `ZonedDateTime` them it would require changes in v1 codebase.
Maybe we could comeback it once we removed v1 from our codebase. If its ok with you I could create Jira for it so that this work would not lost.